view/frontend/web/instantsearch.js

```diff
@@ -96,6 +95,60 @@ requirejs(['algoliaBundle','Magento_Catalog/js/price-utils'], function(algoliaBu
 					$('.algolia-instant-replaced-content').hide();
 					$('.algolia-instant-selector-results').show();
 				}
+			},
+			routing : {
+        stateMapping: {
```

Comment:
Could you fix the indentation? In JS files we are stuck with `tabs` :/ I'm aware of the inconsistency and eventually we'll transform it to spaces, but I'd rather do it in a separate branch to not to pollute a feature branch.